packages/config/config/devices/0x0234/zdb5100.json

```
@@ -4,824 +4,494 @@
+			"unit": "1.0 seconds",
+			"valueSize": 2,
+			"maxValue": 43200,
+			"defaultValue": 300,
+			"allowManualEntry": true
+		},
+		"18": {
+			"$import": "templates/logic_group_template.json#multilevel_switch_set",
+			"label": "Multi-Level Switch Set values for pushbutton 1"
+		},
+		"19": {
+			"$import": "templates/logic_group_template.json#binary_switch_set",
+			"label": "Binary Switch Set support for pushbutton 1"
+		},
 		"20": {
-			"label": "Pushbutton 1 LED indication",
-			"description": "This parameter specifies how the LED indication is controlled.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 7,
-			"defaultValue": 7,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Internal LED control is disabled, only external.",
-					"value": 0
-				},
-				{
-					"label": "LED follows the switch status.",
-					"value": 1
-				},
-				{
-					"label": "LED follows the switch inverse status",
-					"value": 2
-				},
-				{
-					"label": "Same as 1",
-					"value": 3
-				},
-				{
-					"label": "Same as 2",
-					"value": 4
-				},
-				{
-					"label": "Follow internal dimmer",
-					"value": 5
-				},
-				{
-					"label": "Follow internal dimmer inverse",
-					"value": 6
-				},
-				{
-					"label": "On for 5 sec",
-					"value": 7
-				}
-			]
+			"$import": "templates/logic_group_template.json#pushbutton_led_indicator"
 		},
 		"21": {
-			"label": "Colour commands for pushbutton 1 LED indication",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 2,
-			"defaultValue": 0,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Direct control",
-					"value": 0
-				},
-				{
-					"label": "Color for OFF state",
-					"value": 1
-				},
-				{
-					"label": "Color for ON state",
-					"value": 2
-				}
-			]
+			"$import": "templates/logic_group_template.json#color_command_pushbutton",
+			"label": "Colour Commands For Pushbutton 1 LED Indication"
 		},
-		"22": {
-			"label": "ON indication RGB-saturation for pushbutton 1",
+		"22[0xff]": {
+			"$import": "templates/logic_group_template.json#led_indicator",
+			"label": "Pushbutton 1 LED indicator ON Red",
 			"valueSize": 4,
-			"minValue": 0,
-			"maxValue": 4294967295,
-			"defaultValue": 32512,
-			"unsigned": true,
-			"readOnly": false,
-			"writeOnly": false,
+			"defaultValue": 0
+		},
+		"22[0xff00]": {
+			"$import": "templates/logic_group_template.json#led_indicator",
+			"label": "Pushbutton 1 LED indicator ON Green",
+			"valueSize": 4,
+			"defaultValue": 0
+		},
+		"22[0xff0000]": {
+			"$import": "templates/logic_group_template.json#led_indicator",
+			"label": "Pushbutton 1 LED indicator ON Blue",
+			"valueSize": 4,
+			"defaultValue": 127
+		},
+		"22[0xff000000]": {
+			"$import": "../templates/master_template.json#base_0-255_nounit",
+			"label": "LED Time for pushbutton 1 blinking",
+			"unit": "x0.1 sec",
```

```suggestion
			"unit": "0.1 seconds",
```
Please check the style guide so your units are consistent

---

resolved in the latest update